my-uw/index.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: my-uw-template
+title: MyUW Style Guide
+slug: my-uw
+---
+<p class="lead">These are the styles for the redesign of MyUW. You can view the beta redesign by logging in with your NetID <a href="https://my.wisc.edu/portal/Login?profile=bucky" target="blank">here</a>. This page is a work in progress. These styles are all namespaced under the class 'my-uw', which affects everything outside the MyUW frame. The frame consists of the sidebar, header, and footer.</p>
```

Comment:
Is this right? 

> These styles are all namespaced under the class 'my-uw', which affects everything *outside* the MyUW frame.

Maybe I'm not getting what the MyUW frame is, but should that say *inside* instead? Probably not, but just checking.